parent/pom.xml

```diff
@@ -138,14 +138,28 @@
     <ignored.changes.file>ignored-changes.json</ignored.changes.file>
 
     <!--
-      you can use the following to disable some or all tests:
+      you can use the following to disable some or all goals:
         - skipTests will skip all tests
         - skipITs will skip integration tests
         - skipUTs will skip unit tests
+        - skipChecks will non-test checks, such as licensing, checkstyle, backwards compat, etc.
       -->
     <skipTests>false</skipTests>
     <skipUTs>${skipTests}</skipUTs>
     <skipITs>${skipTests}</skipITs>
+
+    <!--
+      you can use the following to disable specific checks. the approach taken is to use a single
+      new property, skipChecks, which is the default value for pre-defined properties such as
+      checkstyle.skip. that way, you can use skipChecks, but you can also still use checkstyle.skip
+      if you only want to disable checkstyle
+      -->
+    <skipChecks>false</skipChecks>
+    <checkstyle.skip>${skipChecks}</checkstyle.skip>
+    <revapi.skip>${skipChecks}</revapi.skip>
+    <license.skip>${skipChecks}</license.skip>
+    <enforcer.skip>${skipChecks}</enforcer.skip>
+    <mdep.analyze.skip>${skipChecks}</mdep.analyze.skip>
```

Comment:
👍 Love this! Can we add the fmt-maven-plugin also?

Reply:
Good catch :+1: